packages/tools/accessibility/README.md

```diff
@@ -0,0 +1,88 @@
+This package renders accessibility tree (html twins of the scene objects) for 3D content in DOM, thus the 3D content is compatible with screen readers and keyboard navigation.
+
+## What is Screen Reader?
+
+[Screen reader](https://en.wikipedia.org/wiki/Screen_reader) is an assistive technology for blind or low vision people to interact with digital content. For users with blindness or low vision, they can't rely 100% on visual interface. Depending on the degree of visual disabilities, some of they don't even use a monitor display or mouse when they use computers. So, for this kind of user, they use screen reader, a software, to read what are displayed visually on the screen, through speech, or through braille output. They use keyboard to tell the screen reader what to read and interact with the computer. Here's an example video of how a user uses screen reader: [Screen Reader Demo - YouTube](https://www.youtube.com/watch?v=q_ATY9gimOM&ab_channel=SLCCUniversalAccess).
+
+There are different screen reader applications that a user can choose, including OS built-in screen readers like Windows's "Narrator", MacOS and iOS's "VoiceOver", and 3-rd party screen readers like JAWS, NVDA.
+
+Nowadays 2D pages on web generally have good accessibility support to screen reader users, because it can understand different HTML elements. But 3D applications like webGL applications is not. This is because objects in webGL applications are rendered in a <canvas> element. When screen reader read the page, it will only read: "*Image*", but can't interpret any objects inside the scene. If we do not deal with it correctly, users who are blind or low vision will have difficulty using the application.

```

Comment:
"But 3D applications like webGL applications is not." should be:
"But 3D applications like webGL applications **do** not." for subject/predicate agreement